### PR TITLE
fix mongoose issue

### PR DESCRIPTION
**Pull request by peerwit**

### CODE REVIEW

File: server/api/amazonCart/product.controller.js [hunk 1]
```diff
@@ -30,16 +31,19 @@ exports.createCart = function(req, res, next) {
...
       if (req.user) {
         var user = req.user;
         user.cart = cart;
         user.cart.items = {};
         user.cart.items[req.body.id] = 1;
         user.cart.Quantity = 1;
-        user.ASIN2CartItemId = user.ASIN2CartItemId || {};
-        user.ASIN2CartItemId[req.body.id] = cart.CartItems[0].CartItem[0].CartItemId[0];
-        user.save(function(err) {
+        user.cart.aaa = {};
```
Comment (scottrice10) [inline server/api/amazonCart/product.controller.js]: What does 'aaa' stand for?

File: server/auth/auth.service.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ function isAuthenticated() {
       User.findById(req.user._id, function (err, user) {
         if (err) return next(err);
         if (!user) return res.send(401);
-
+        console.log(user, "...from Mongo");
         req.user = user;
```
Comment (scottrice10) [inline server/auth/auth.service.js:36]: Clean up console.log's

**Comment by tliber** [comment]
:+1:  please change 'aaa' to reasonable variable

**Comment by peerwit** [comment]
I've changed the name to ASIN2CART

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**